app/feedperson/tests.py

```diff
@@ -14,16 +14,29 @@ def test_str_value(self):
     feed_person = FeedPerson.objects.get(id=1)
     self.assertEqual(feed_person.name, str(feed_person))
 
-  def test_load_feed_people_removes_original_data(self):
-    feed_person = FeedPerson.objects.get(id=1)
+class LoadFeedTest(TestCase):
+  
+  @classmethod
+  def setUpTestData(cls):
     with disable_auto_indexing():
       load_feed_people()
-    self.assertEqual(FeedPerson.objects.filter(eppn=feed_person.eppn).count(), 0)
 
   def test_load_feed_people_imports_western_ave_people(self):
-    with disable_auto_indexing():
-      load_feed_people()
     western_ave_people = filter(lambda person: "Western Ave" in person.location, FeedPerson.objects.all())
     non_western_ave_people = filter(lambda person: "Western Ave" not in person.location, FeedPerson.objects.all())
     self.assertTrue(len(list(western_ave_people)) > 0)
     self.assertEqual(len(list(non_western_ave_people)), 0)
+
+  def test_load_feed_people_regex_removes_all_SciEnG(self):
+    scieng_people = filter(lambda person: "Sci&Eng" in person.location, FeedPerson.objects.all())
+    self.assertTrue(len(list(scieng_people)) == 0)
```

Comment:
Seems like it would be better to use `self.assertEqual(len(list(scieng_people)), 0)` here, as in the test above. That's generally how we'd do it in our JS projects, as the equality assertions tend to show better output in the case of failure (i.e., "Expected 0, got 100" instead of "Expected true, got false").